TimSort/timsort.c

```diff
@@ -0,0 +1,117 @@
+//GSSoC
+//TimSort
+
+#include<stdio.h>  
+
+int minimum(int a, int b)  
+{  
+        if(a<b)  
+        return a;   
+        else   
+        return b;   
+}  
+ 
+//MERGE SORT FUNCTION :   
+void merge(int array[], int left, int mid, int right)  
+{  
+        int LeftLength = mid - left + 1, RightLength = right - mid;  
+        int Left[LeftLength], Right[RightLength];  
```

Comment:
either use hungarian notation or camel case

Reply:
I will use the camelCase.... thankYou!